pom.xml

```diff
@@ -12,6 +12,13 @@
     <version>1.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
+    <licenses>
+        <license>
+          <name>Vaadin Commercial License</name>
```

Comment:
Shouldn't this be 'Vaadin Commercial License and Service Terms'?

Reply:
Yes it should! Fixed, thanks.